src/us/kbase/genbank/GbkQualifier.java

```diff
@@ -5,29 +5,32 @@
  * @author rsutormin
  */
 public class GbkQualifier extends GbkElement {
-	boolean is_text = false;
-	boolean is_closed = false;
-	public GbkQualifier(int line_num,String type,String value) {
-		super(line_num,type,value);
+    boolean is_text = false;
+    boolean is_closed = false;
+    public GbkQualifier(int line_num,String type,String value) {
+	super(line_num,type,value);
+    }
+    public void close() {
+	is_closed = true;
+	String val = value.toString();
+	if (val.startsWith("\"")) {
+	    if (val.endsWith("\"")) {
+		is_text = true;
+		value.deleteCharAt(0);

```

Comment:
Future point to discuss, we should be compiling a list of these "corrections", what line numbers they represent, etc, and storing that in provenance.